File: src/CoursTech.Domain/Domain/Entities/Course.cs
namespace Domain.Entities
{
    public class Course
    {
        public Guid CourseId { get; set; }
        public string Title { get; set; }
        public TimeSpan Duration { get; set; }
        public string Description { get; set; }
        
        public Guid InstructorId { get; set; }
        public Instructor Instructor { get; set; }

        public int IndustryId { get; set; }
        public Industry Industry { get; set; }

        public ICollection<Enrollment> Enrollments { get; set; }
    }
}

File: CoursTech.WebUI/CoursTech/src/app/models/Course.ts
import { Enrollment } from "./Enrollment";
import { Industry } from "./Industry";
import { Instructor } from "./Instructor";

export interface Course {
    courseId: string;
    title: string;
    duration: string;
    date: string;
    description: string;
    imageName: string;
    image: File | null;
    instructorId: string;
    instructor: Instructor | null;
    industryId: number;
    industry: Industry | null;
    enrollments: Enrollment[] | null;
}